File: java/conundrum.java
import java.util.Scanner;

public class conundrum {
	static final String letter = "PER";

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		String chipertext = scan.next();
		int index = 0, count = 0;
		for (char c : chipertext.toCharArray())
			if (c != letter.charAt(index++ % 3))
				count++;
		System.out.println(count);
		scan.close();
	}
}

File: ts/src/conundrum.ts
import readline = require('readline')

function getDays (cipher: string): number {
  let days = 0
  cipher.split('').forEach((char, index) => {
    if (char !== letter.charAt(index % letter.length)) {
      days++
    }
  })
  return days
}

const letter = 'PER'

readline
  .createInterface({
    input: process.stdin,
    output: process.stdout
  })
  .on('line', (line: string) => {
    console.log(getDays(line))
  })
  .on('close', () => {
    process.exit(0)
  })
